plugin/xcodebuild.vim

```diff
@@ -3,13 +3,27 @@ command! XTest call <sid>test()
 
 let s:plugin_path = expand('<sfile>:p:h:h')
 
+function! s:bin_script(name)
+  return s:plugin_path . '/bin/' . a:name
+endfunction
+
+function! s:cli_args(...)
+  let cli_args = ''
+
+  for cli_arg in a:000
```

Comment:
you telling me I should be using functors, gabe?

also, does `join` add a preceding space? I'm assuming not. So this would _probably_ have to be

```vim
return ' ' . join(map(a:000, shell escape(v:val)))
```

right?